src/Mod/PartDesign/Gui/TaskPadParameters.cpp

```diff
@@ -201,7 +204,8 @@ void TaskPadParameters::updateUI(int index)
     bool isLengthEdit2Visable = false;

```

Comment:
While you're making edits here anyway, I think it would be a good idea to correct this misspelling (visable->visible).

Reply:
I changed their name now.